conduct.md

```diff
@@ -7,16 +7,282 @@
+  * Teams will bring up problems and errors that occurred, or what worked well.
+  * After finishing a project teams should be assigned to look after the program.
+  * Work as a team to fix problems.
+  * Use proper communication to talk to one another.
+
+* **Unacceptable (U)**
+  * Do not blame anyone for errors or problems
+  * Do not work alone use your team members as a guide
+  * Do not attempt to change things without talking to others
+
+## Conflicts
+
+In the conflicts section of the project, we deal with conflicts in a
+professional manner by communicating any type of issue/ conflict with any of the
+team leaders, technical leads, or the CEO directly.
+
+* **Acceptable Behavior (C)**
+  * Team Members shall:
+    * Not unduly use their position/ influence in this project for personal
+    benefit or to benefit relatives or close associates.
+    * Report a conflict or potential conflict with either a team leader,
+    technical lead or the CEO directly as soon as it arises or at the nearest
+    availability.
+  * Team Leads shall:
+    * Be open and respect both sides looking for the most beneficial option
+    towards the project.
+    * Make decisions with the best interest of the team as a whole upon proper
+    discussion.
+* **Unacceptable Behavior (U)**
+  * Team Members shall NOT:
+    * Bring about personal interests that will bring about conflicts with
+    those of the team as a whole.
+    * Move forward on a conflict that has not yet been discussed or resolved
+    to prevent even further conflicts to arise.
+  * Team Leads shall NOT:
+    * Disregard any conflict that has been brought to their attention.
+    * Use their position/ influence to make any type of decision that is not
+    beneficial to the team as a whole.
+
+## Infractions
+
+An ***infraction*** shall hereby be defined as a penalty given as a result of a
+violation of the above outlined Conduct Guide by any individual who has agreed
+to adhere to the Conduct Guide via digital signature below. The infractions for
+any individual shall be recorded and managed by the instructor.
+
+A *violation* of the Conduct Guide is defined as any behavior that results in receiving
+a "U" (Unacceptable) for any of the above outlined (sub)sections.
+
+Infractions shall be imposed at three different levels of severity:
+
+* **Level I**: A ***Level I infraction*** is defined as a formally documented
+ violation of the above outlined Conduct Guide. A Level I infraction may be
+ suggested to the instructor by a team or project leader for any given project.
+ All Level I infractions will be reset weekly.
+* **Level II**: A ***Level II infraction*** will be given to an individual who
+ has accumulated three Level I infractions. Upon obtaining a Level II infraction,
+ the individual will no longer be eligible to receive a check for the week. All
+ Level II infractions will be reset weekly.
+* **Level III**: A ***Level III infraction*** will be given to an individual who
```

Comment:
One thing isn't clear here -- if a Level II infraction is reached, do the same offenses also count towards a Level III? That should be explicitly stated if so. I would also suggest Level III infractions requiring two Level II's, perhaps with a change to the 10% as well to compensate.

Reply:
From `conduct.md`:

> A ***Level II infraction*** will be given to an individual who has accumulated three Level I infractions.
> ...
> A ***Level III infraction*** will be given to an individual who has accumulated five Level I infractions.

The intent is for the Level I infractions accumulated each week to count towards either of the higher level infractions. That is to say, yes, the offenses count towards both Level II and Level III infractions. The rationale being that, since Level I infractions are reset weekly, it should be fairly difficult for an individual to accumulate so many in the span of only seven days. 

Would wording along the lines of: 

> A ***Level II infraction*** will be given to an individual who has accumulated three Level I infractions in a single week.
> ...
> A ***Level III infraction*** will be given to an individual who has accumulated five Level I infractions in a single week.

Be a better approach to make this more clear? 

And to address the suggestion of having a Level III infraction require two Level II infractions, we considered this and decided against it for two reasons:
* We feared confusion or ambiguity when counting infractions. We didn't want a student having to "calculate" the number of Level I infractions and Level II infractions to figure out where they stand in terms of these infractions. We felt that just keeping track of the Level I infractions as a flat number and receiving Level II and Level III infractions at set numbers of Level I infractions each week would be easier for both the student and the instructor to keep track of.
* We also decided that six Level I infractions for one Level III infraction in a week was too lenient. While the difference is only one low-level infraction, we saw no reason to anticipate students breaching the Conduct Guide five times in the span of only one week. We believe that the punishment of 10% of the overall grade for the project, as well as the number of infractions in a week required to get to that point, is both fair and reasonable for helping ensure that we are productive as a class. If this turns out not to be the case, I think it can be opened for discussion later, but I personally struggle to foresee any individual breaking the rules of the Conduct Guide five times in one week multiple times (or, really, once for that matter).

If you think this rationale should be further discussed, I invite you to open an Issue on the fork with the discussion tag to bring more people into the conversation. I want everyone to be on the same page with regard to how strict the infractions are, as we are all still students first and this is a critical component for all of our grades.

Reply:
I think that should be a little clearer. You might want to explicitly say "Level I infractions counting toward a Level II infraction can also count towards a Level III infraction", since I think that might be the only point of confusion left. Your justification for keeping the flat structure sounds good, though!

Reply:
Alright, I added the clarification statement to the following paragraph below the list of infraction levels. I am currently awaiting approval for the Pull Request on Aubrey's fork, which should come shortly. 

Thanks for your suggestions, @Michionlion!